db/flush_job.cc

```diff
@@ -960,7 +960,8 @@ Status FlushJob::WriteLevel0Table() {
 
       const std::string* const full_history_ts_low =
           (full_history_ts_low_.empty()) ? nullptr : &full_history_ts_low_;
-      const ReadOptions read_options(Env::IOActivity::kFlush);
+      ReadOptions read_options(Env::IOActivity::kFlush);
+      read_options.rate_limiter_priority = io_priority;
```

Comment:
Nit: On line 864, io_priority is calculated using a call to `GetRateLimiterPriorityForWrite()`. Can we change the name to just `GetRateLimiterPriority()`?

Reply:
Fixed